server/datastore/mysql/hosts.go

```diff
@@ -808,7 +809,7 @@ func (ds *Datastore) ListHosts(ctx context.Context, filter fleet.TeamFilter, opt
     h.osquery_host_id,
     h.created_at,
     h.updated_at,
-    h.detail_updated_at,
+    COALESCE(h.detail_updated_at, h.created_at) as detail_updated_at,
```

Comment:
Let's revert as this is not necessary.

Reply:
As discuss on a previous call `COALESCE(h.detail_updated_at, h.created_at)` is not necessary (on all occurrences in this PR) because currently `h.detail_updated_at` cannot be `NULL` (because of how hosts enroll).